main.js

```diff
@@ -0,0 +1,145 @@
+const player = (name, sym) => {
+  const getName = () => name;
+  const getSymbol = () => sym;
+
+  return { getSymbol, getName };
+};
+
+const gameCycleController = (() => { // eslint-disable-line no-unused-vars
+  let gameBoardArray = [];
+  const statusAlert = document.getElementById('statusAlert');
+  let player1;
+  let player2;
+  let currentPlayer;
+  let count = 1;
+  const win = () => {
+    statusAlert.innerText = `${currentPlayer.getName()} won this game`;
+  };
+  const draw = () => {
+    statusAlert.innerText = 'Not bad it is a draw';
+  };
+  const gameStatus = () => {
+    /* eslint-disable */
+    switch (true) {
```

Comment:
Hi, we have fixed other thing but this is difficult because this is how we test for winning conditions and draw were applicable. Basically, each case test for each possible winning condition and calls the win function, so it's not a repetition in that sense. I don't see a way we can simplify this.

Reply:
Update we did a little simplification, basically created a function to handle all the work. Good insight :smile: